res/openvpn-gui-res-it.rc

```diff
@@ -485,45 +499,47 @@ Le operazioni che richiedono accesso amministrativo potrebbero non funzionare."
-    IDS_ERR_CONN_SCRIPT_TIMEOUT "Il valore di registro ""connectscript_timeout""deve essere un numero tra 0 and 99."
+    IDS_ERR_READING_REGISTRY "Errore nella lettura del valore del registro (HKLM\\SOFTWARE\\OpenVPN)."
+    IDS_ERR_PASSPHRASE_ATTEMPTS "Il valore del registro ""passphrase_attempts"" deve essere un numero tra 1 e 9."
+    IDS_ERR_CONN_SCRIPT_TIMEOUT "Il valore del registro ""connectscript_timeout""deve essere un numero tra 0 and 99."
     IDS_ERR_DISCONN_SCRIPT_TIMEOUT "Il valore di registro ""disconnectscript_timeout"" deve essere un numero tra 1 and 99."
     IDS_ERR_PRECONN_SCRIPT_TIMEOUT "Il valore di registro ""preconnectscript_timeout"" deve essere un numero tra 1 and 99."
     IDS_ERR_CREATE_REG_KEY "Errore nella creazione della chiave HKLM\\SOFTWARE\\OpenVPN-GUI."
-    IDS_ERR_OPEN_WRITE_REG "Errore nell'apertura del registro in scrittura. Puoi far partire questa applicazione \
+    IDS_ERR_OPEN_WRITE_REG "Errore nell'apertura del registro in scrittura.\nPuoi far partire questa applicazione \
 una volta che l'amministratore ha aggiornato il registro."
-    IDS_ERR_READ_SET_KEY "Errore nella lettura e modifica della chiave di registro ""%ls""."
-    IDS_ERR_WRITE_REGVALUE "Errore nella scrittura del valore di registro ""HKEY_CURRENT_USER\\%ls\\%ls""."
+    IDS_ERR_READ_SET_KEY "Errore nella lettura e modifica della chiave del registro ""%ls""."
+    IDS_ERR_WRITE_REGVALUE "Errore nella scrittura del valore del registro ""HKEY_CURRENT_USER\\%ls\\%ls""."
 
     /* importation */
     IDS_ERR_IMPORT_EXISTS "Esiste già una configurazione con il nome ""%ls""."
-    IDS_ERR_IMPORT_FAILED "Errore nell'importazione del file. Non ho potuto creare il percorso seguente:\n\n\
+    IDS_ERR_IMPORT_FAILED "Errore nell'importazione del file.\nImpossibile creare il seguente percorso:\n\n\
 %ls\n\nControlla di avere i permessi giusti."
-    IDS_NFO_IMPORT_SUCCESS "File importato con successo."
-    IDS_NFO_IMPORT_OVERWRITE "A config named ""%ls"" already exists. Do you want to replace it?"
-    IDS_NFO_IMPORT_SOURCE "Do you want to import the profile <%ls>?"
-    IDS_ERR_IMPORT_SOURCE "Cannot import file <%ls> as it is already in the global or local config directory"
-    IDS_ERR_IMPORT_ACCESS "Cannot import <%ls> as it is missing or not readable"
+    IDS_NFO_IMPORT_SUCCESS "File importato correttamente."
+    IDS_NFO_IMPORT_OVERWRITE "Esiste già un file configurazione ""%ls"".\nVuoi sostituirlo?"
+    IDS_NFO_IMPORT_SOURCE "Vuoi importare il profilo <%ls>?"
+    IDS_ERR_IMPORT_SOURCE "Impossibile importare il file <%ls> poiché è già presente nella cartella di configurazione globale o locale"
+    IDS_ERR_IMPORT_ACCESS "Impossibile importare <%ls> in quanto mancante o non leggibile"
 
     /* save/delete password */
-    IDS_NFO_DELETE_PASS "Premi OK per cancellare le password salvate per la configurazione ""%ls"""
+    IDS_NFO_DELETE_PASS "Seleziona 'OK' per cancellare le password salvate per la configurazione ""%ls"""
```

Comment:
When it's a button do you really say "seleziona"? might "Fai click su" be better?